packages/context/src/lib/share-map/share-map/share-map-api.component.ts

```diff
@@ -33,7 +33,9 @@ export class ShareMapApiComponent implements OnInit {
   ngOnInit(): void {
     this.auth.authenticate$.subscribe((auth) => {
       const decodeToken = this.auth.decodeToken();
-      this.userId = decodeToken.user ? decodeToken.user.id : undefined;
+      this.userId = decodeToken?.user
+        ? decodeToken.user.id.toString()
+        : undefined;
```

Comment:
On pourrait simplifier:
```suggestion
      this.userId = decodeToken?.user?.id.toString();
```